stable/openvpn/values.yaml

```diff
@@ -76,7 +76,7 @@ openvpn:
   # keystoreSecret:
   # secret with openvpn certificates. If specified, certificates are taken from the secret
   # create secret with such command:
-  #    kubectl create secret generic openvpn-keystore-secret --from-file=./server.key --from-file=./ca.crt --from-file=./server.crt --from-file=./dh.pem
+  #    kubectl create secret generic openvpn-keystore-secret --from-file=./server.key --from-file=./ca.crt --from-file=./server.crt --from-file=./dh.pem --from-file=./crl.pem  [--from-file=./ta.key]
```

Comment:
Have you tested what happens if this file is not provided (doesn’t exist in the secret)? Is the crl mandatory?

Reply:
I added a value openvpn.useCrl, to control whether to use/create crl.pem or not. By default it's 'false'. The patch also solves https://github.com/helm/charts/issues/14824

Reply:
crl became mandatory in openvpn 3.12.5 (commit https://github.com/helm/charts/commit/100d5a156b00d06214bff8eba8b259cce2daa63c#diff-426bf77ebc46805831ad90724aa62549), due to crl-verify

Reply:
Ah great, I didn’t realise that had been introduced - thanks for fixing it!